stdlib/Mmap/src/Mmap.jl

```diff
@@ -155,6 +164,8 @@ privileges are required to grow the file.
 The `shared` keyword argument specifies whether the resulting `Array` and changes made to it
 will be visible to other processes mapping the same file.
 
+The `exec` keyword argument specifies whether the underlying mmap data will be executable.
```

Comment:
```suggestion
The `exec` keyword argument specifies whether the underlying mmap data will be executable.

!!! note
    On MacOS `exec=true` implies `shared=false`, because each thread has its own access permissions to `mmap` regions. 

```